src/csvcubed/utils/validations.py

```diff
@@ -226,6 +226,26 @@ def validate(
     return validate
 
 
+def all_of(*conditions: ValidationFunction) -> ValidationFunction:
+    """
+    This function will validate if all arguments provided are instances of the types
```

Comment:
Not 100% sure about this comment. It validates that a property matches all of the conditions provided.

Reply:
done